src/ng-select/lib/ng-select.component.ts

```diff
@@ -493,7 +493,7 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
     };
 
     get showAddTag() {
-        if (!this.searchTerm) {
+        if (!this.searchTerm || (this.searchTerm && !this.searchTerm.trim())) {
```

Comment:
you can trim before and keep the condition same
```js
cont term = this.searchTerm && this.searchTerm.trim()
if (!term) {
...
```

Reply:
@varnastadeus Due to some issues, I had to some force push which result of closing this PR. Created another PR #1383 with the mentioned changes. Please review.